src/ingest_llm_as/services/omega_ingest_guardian.py

```
@@ -0,0 +1,241 @@
+                "events": events,
+                "metadata": {
+                    "version": self.version,
+                    "processed_at": datetime.now(timezone.utc).isoformat(),
+                    "entity_count": len(entities),
+                    "relationship_count": len(relationships),
+                    "event_count": len(events),
+                },
+            }
+
+        except ET.ParseError as e:
+            self.logger.error(f"Failed to parse POML data: {str(e)}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)